### PR TITLE
Support for BottomNavigationView added

### PR DESCRIPTION
**Pull request by Alviere**

### CODE REVIEW

File: kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt [hunk 1]
```diff
@@ -515,6 +518,31 @@ interface ImageViewAssertions : BaseAssertions {
...
+    fun hasSelectedItem(id: Int) {
+        view.check { view, notFoundException ->
+            if (view is BottomNavigationView) {
+                if (view.selectedItemId != id) {
+                    throw AssertionError("Expected selected item id is $id," +
+                            " but actual is ${ view.selectedItemId }")
+                }
+            } else {
+                notFoundException?.let {
+                    throw AssertionError(notFoundException)
```
Comment (Judrummer) [inline kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt]: Can we use ->  throw AssertionError(it)

Comment (Alviere) [reply kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt]: Yeah, sure

Comment (Alviere) [reply kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt]: Done

File: kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt [hunk 2]
```diff
@@ -5,6 +5,8 @@ import android.graphics.drawable.Drawable
 import android.support.annotation.ColorRes
 import android.support.annotation.DrawableRes
 import android.support.annotation.StringRes
+import android.support.design.widget.BottomNavigationView
+import android.support.test.espresso.NoMatchingViewException
```
Comment (VerachadW) [inline kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt]: I didn't see the usage of this import

Comment (Alviere) [reply kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt]: Probably it got imported when I was auto-implementing anonymous object of class `ViewAssertion`, then converted to lambda and forgot to remove it.

**Comment by Alviere** [comment]
@VerachadW @Judrummer everything is fixed

**Pull request merged**